filetracker/client/client.py

```diff
@@ -194,9 +199,12 @@ def get_stream(self, name, force_refresh=False, serve_from_cache=False):
                     return self.local_store.get_stream(name)
```

Comment:
Nie wiem na ile możliwe jest, żeby get_stream rzuciło wyjątek tuż po dodaniu przez add_stream.
Ale jeśli jest to możliwe, to tu jest ten sam problem co w put_file - plik w cache bez locka.